src/features/bank-request-info/bank-services.ts

```diff
@@ -14,6 +14,19 @@ export const services: Service[] = [
       "Diamond (Price on Jewelry Spreadsheet, _North Freeport Bank_)",
     ],
   },
+  {
+    title: "OT Hammer Event Scheduling",
+    icon: Icon.Sold,
+    requestFormats: ["OT Hammer Event", "OT Hammer Event: Date, Time (include timezone)"],
```

Comment:
This field is communicating to players how they ask for an event. Are these two different ways we want them to ask?

Reply:
yeah, thought they could just do one for requesting an event at the default time and then another if they know that won't work and want to try requesting a diff time.